### PR TITLE
docs: multiple auth roles directive

### PR DESCRIPTION
**Pull request by ninnjak**

### CODE REVIEW

**Review by jonnydgreen — APPROVED** [review]
Amazing! 🚀 LGTM!

File: docs/auth-directive.md [hunk 1]
```diff
@@ -188,3 +188,74 @@ app.register(mercuriusAuth, {
...
+## Mutliple Auth Roles Directive
+
+The auth directive can be modified to allow multiple roles to be passed in.
+
+```js
+'use strict'
+
+const Fastify = require('fastify')
+const mercurius = require('mercurius')
+const mercuriusAuth = require('..')
```
Comment (jonnydgreen) [inline docs/auth-directive.md:201]: Minor nit to improve the docs

```suggestion
const mercuriusAuth = require('mercurius-auth')
```

Comment (ninnjak) [reply docs/auth-directive.md]: I've made the change you suggested and also fixed the lint errors

**Comment by jonnydgreen** [comment]
All looks good - great work! The CI error for windows 14 (now EOL) is unrelated to this PR so will merge this and in a follow-up PR, drop support for Node.js 14 (and add v20 support) :)

**Pull request merged**